### DISCUSSION TITLE
8월 30일 일기

### DISCUSSION THREAD

**Discussion by iamyanghee** [Melon-Shake/API#43] [team]
칸반은 주로 짧게 태스크만 얘기해야하고
기분이 어떻다 컨디션이 어떻다 
얼굴 보고 말하기도 뭐하고 
그래서 난 일기 괜찮은 것 같은데

다들 어땠나 모르겠다
오늘도 진행시켜!

서로 챙겨주자~ 😊
![1693347285411](https://github.com/Melon-Shake/main_melonshake/assets/131243101/541a6216-c6bb-40b3-a9b9-35b3738114aa)

**Comment by iamyanghee** [comment]
@iamyanghee 

네시에 눈 떠져서 일찍 나와봄
상쾌한듯 멍한듯

**Reply by Lucete28** [reply]
코딩하기 딱 좋은 날이네요

**Comment by bksooon** [comment]
오늘 내 아침기분은 잠을 못자서 0.1이야

**Reply by Lucete28** [reply]
내가 가서 0.1인거 아닐까요

**Comment by jungssg** [comment]
바지가 없어진거같다...
말이되나?

**Reply by Lucete28** [reply]
다행이 입고는 오셧네요

**Comment by json9101** [comment]
실제기업에서 할만한 프로젝트는 처음이니 시행착오는 당연히 있을수 밖에 없다고 감안하고 다시 힘내서 합시다

**Reply by Lucete28** [reply]
화이팅

**Comment by Lucete28** [comment]
pd 마지막 출근 ㅠㅠ
커피 마셔야징

**Comment by mews2000** [comment]
오늘 내로 데모 완성가능했으면 좋겠다

**Comment by bksooon** [comment]
오늘 퇴근 기분 점수: 3.0 (아침: 0.1)

오늘은 가슴운동을 하는날이다 

그래서 그런지 기분도 좋고 타팀으로부터 정보도 받아서 실행가능성이 보이니 활력이 돈다. 소확행.....

**Comment by Lucete28** [comment]
개인적으로 진도를 만족할 만큼 빼서 좋았다
양희누나가 마지막이라 서운할까봐 많이 물어봐줘서 좋았다
마지막이라 너무 아쉽다. 
양꼬치 맛있겠다

**Comment by iamyanghee** [comment]
얘네들 왜 이럴까
